eos/form-factors/parametric-bfw2010-impl.hh

```diff
@@ -0,0 +1,808 @@
+/* vim: set sw=4 sts=4 et foldmethod=syntax : */
+
+/*
+ * Copyright (c) 2022 Méril Reboud
+ *
+ * This file is part of the EOS project. EOS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License version 2, as published by the Free Software Foundation.
+ *
+ * EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#ifndef EOS_GUARD_EOS_FORM_FACTORS_PARAMETRIC_BFW2010_IMPL_HH
+#define EOS_GUARD_EOS_FORM_FACTORS_PARAMETRIC_BFW2010_IMPL_HH 1
+
+#include <eos/form-factors/parametric-bfw2010.hh>
+#include <eos/maths/power-of.hh>
+#include <eos/utils/diagnostics.hh>
+
+#include <numeric>
+
+namespace eos
+{
+    template<typename Process_>
+    BFW2010FormFactors<Process_, PToV>::BFW2010FormFactors(const Parameters & p, const Options &) :
+        _a_A0{  UsedParameter(p[_par_name("A0", 0)],  *this),
+                UsedParameter(p[_par_name("A0", 1)],  *this),
+                UsedParameter(p[_par_name("A0", 2)],  *this),
+                UsedParameter(p[_par_name("A0", 3)],  *this),
+                UsedParameter(p[_par_name("A0", 4)],  *this)
+        },
+        _a_A1{  UsedParameter(p[_par_name("A1", 0)],  *this),
+                UsedParameter(p[_par_name("A1", 1)],  *this),
+                UsedParameter(p[_par_name("A1", 2)],  *this),
+                UsedParameter(p[_par_name("A1", 3)],  *this),
+                UsedParameter(p[_par_name("A1", 4)],  *this)
+        },
+        _a_V{   UsedParameter(p[_par_name("V", 0)],   *this),
+                UsedParameter(p[_par_name("V", 1)],   *this),
+                UsedParameter(p[_par_name("V", 2)],   *this),
+                UsedParameter(p[_par_name("V", 3)],   *this),
+                UsedParameter(p[_par_name("V", 4)],   *this)
+        },
+        _a_T1{  UsedParameter(p[_par_name("T1", 0)],  *this),
+                UsedParameter(p[_par_name("T1", 1)],  *this),
+                UsedParameter(p[_par_name("T1", 2)],  *this),
+                UsedParameter(p[_par_name("T1", 3)],  *this),
+                UsedParameter(p[_par_name("T1", 4)],  *this)
+        },
+        _a_T23{ UsedParameter(p[_par_name("T23", 0)], *this),
+                UsedParameter(p[_par_name("T23", 1)], *this),
+                UsedParameter(p[_par_name("T23", 2)], *this),
+                UsedParameter(p[_par_name("T23", 3)], *this),
+                UsedParameter(p[_par_name("T23", 4)], *this)
+        },
+        _a_A12{ UsedParameter(p[_par_name("A12", 1)], *this),
```

Comment:
Why is A_12 not ordered between A_1 and V?

Reply:
Because `A12` and `T2` are the two form-factors that only have 4 parameters instead of 5 due to end-point relations